service/rpc2/server.go

```diff
@@ -62,6 +62,12 @@ type RestartIn struct {
 	// Position to restart from, if it starts with 'c' it's a checkpoint ID,
 	// otherwise it's an event number. Only valid for recorded targets.
 	Position string
+
+	// ResetArgs tell whether NewArgs should take effect.
+	ResetArgs bool
```

Comment:
Do we need this flag, can't we infer by the state of `NewArgs`?

Reply:
Without this flag, we can't distinguish between "reuse the current args" and "use an empty args list".